src/main/java/org/openstreetmap/atlas/checks/base/CheckResourceLoader.java

```diff
@@ -165,6 +208,32 @@ public CheckResourceLoader(final Configuration configuration)
         return checks;
     }
 
+    /**
+     * Loads checks that are enabled by configuration
+     *
+     * @param <T>
+     *            check type
+     * @return a {@link Set} of checks
+     */
+    public <T extends Check> Set<T> loadChecks()
+    {
+        return loadChecks(this::isEnabledByConfiguration, this.configuration);
+    }
+
+    public <T extends Check> Map<String, Set<T>> loadChecksForCountries(
+            final Predicate<Class> isEnabled, final Iterable<String> countries)
```

Comment:
Method name, parameter name for keys are misleading.

Reply:
What would be a better name? This function loads the checks for each country in the string list.

Reply:
I'd suggest keeping it generic and aligned with override definition - groups. Is this method used anywhere though?

Reply:
This version of the function is used within the more Generic ```loadChecksForCountries(Iterable<String> countries)``` which is then used within ```IntegrityCheckSparkJob```